routes/controller/index.js

```diff
@@ -1,20 +1,53 @@
+const firebase = require("../../config/firebase");
 const User = require("../../models/User");
 const { ERROR } = require("../../constants/messages");
-const { OK } = require("../../constants/statusCodes");
+const { OK, BAD_REQUEST } = require("../../constants/statusCodes");
+const { generateToken, verifyToken } = require("../utils/tokens");
+
+async function postLogin(req, res, next) {
+  const { token: idToken } = req.headers;
+
+  try {
+    const {
+      uid, name, picture: profileUrl
+    } = await firebase.auth().verifyIdToken(idToken);
+    const { doc: user } = await User.findOrCreate({
+      uid, name, profileUrl
+    });
+
+    const accessToken = generateToken(user._id);
+    const refreshToken = generateToken(user._id, true);
+
+    res.status(OK);
+    res.json({
+      accessToken,
+      refreshToken
+    });
+  } catch (err) {
+    res.status(BAD_REQUEST);
+    res.json({ error: ERROR.INVALID_TOKEN });
```

Comment:
여기서 에러를 응답을 바로 보내는 것이 아니라, 에러핸들러로 에러를 넘겨주는 것이 좋을 듯 합니다!
```suggestion
    next(createError(BAD_REQUEST, ERROR.INVALID_TOKEN));
```